styles/styles.css

```diff
@@ -53,6 +58,7 @@
   --text-color-light: #4a4a4a;
   --text-color-blue: #043C6D;
   --text-color-gray: #9B9B9B;
+  --text-color-light-gray: rgb(14 31 53 / 50%);
```

Comment:
convert to hex

Reply:
I have checked no diff in using `#0e1f3580`

Reply:
@sneharora1 we can use the nearby grey color for example: existing ` --color-light-gray-dark`